### PR TITLE
Updating READ.me

### PR DESCRIPTION
**Pull request by pkrucz00**
Changes include:
  - new way of initialising the database
  - listing prerequisites
  - updating the year

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,27 +7,43 @@ This repository contains a project created during the Summer Internship of 2022
...
 On the end you can start phoenix app:
+
 ```sh
 mix phx.server
 ```
 
 ## Copyright and License
 
-Copyright 2020, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane_template_plugin)
+Copyright 2023, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane_template_plugin)
```
Comment (LVala) [inline README.md:46]:
```suggestion
Copyright 2022, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane_template_plugin)
```

File: mix.exs [hunk 2]
```diff
@@ -55,7 +55,8 @@ defmodule Membrane.Live.Mixfile do
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
       {:credo, ">= 0.0.0", only: [:dev, :test], runtime: false},
       {:membrane_rtc_engine,
-       github: "membraneframework/membrane_rtc_engine", branch: "synchronize-streams-wallclock"},
+       github: "membraneframework/membrane_rtc_engine",
+       branch: "add-compositor-with-synchronization"},
```
Comment (LVala) [inline mix.exs:59]: Are these dependency changes intentional?

Comment (pkrucz00) [reply mix.exs]: Added by mistake. I've changed it

File: README.md [hunk 3]
```diff
@@ -7,27 +7,43 @@ This repository contains a project created during the Summer Internship of 2022
...
```
Comment (LVala) [inline README.md:46]:
```suggestion
Copyright 2022, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane_template_plugin)
```
Still 2023?

Comment (Rados13) [reply README.md:46]: I agree with Lukasz it should be 2022.

Comment (pkrucz00) [reply README.md]: I also agree it should be 2022

**Review by LVala — APPROVED** [review]

**Review by Rados13 — APPROVED** [review]

**Pull request merged**